uPortal-api/uPortal-api-rest/src/main/java/org/apereo/portal/rest/PrincipalsRESTController.java

```diff
@@ -51,7 +51,7 @@ public void setGroupListHelper(IGroupListHelper groupListHelper) {
      * @throws Exception
      */
     @PreAuthorize(
-            "hasPermission('string', 'REST', new org.apereo.portal.spring.security.evaluator.AuthorizableActivity('UP_PERMISSIONS', 'VIEW_PERMISSIONS'))")
+            "hasPermission('ALL', 'java.lang.String', new org.apereo.portal.spring.security.evaluator.AuthorizableActivity('UP_PERMISSIONS', 'VIEW_PERMISSIONS'))")
```

Comment:
No, it shouldn't.  There's no such permission as `REST`.

Left as `REST`, the API endpoint would continue to be unusable (even for admin).